webapp/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "matterpoll",
+  "version": "1.1.0",
+  "description": "This plugin is webapp for matterpoll plugin.",
```

Comment:
Do we need a description?

Reply:
This desc never be used, so should be removed. Thanks.